source/install/config-cloudfront.rst

```diff
@@ -0,0 +1,47 @@
+.. _config-cloudfront:
+
+Configuring CloudFront to host Mattermost static assets
+=======================================================
+
+1. Create an S3 bucket using your desired domain. In our example it will be mattermost.example.com.
+2. Enable static hosting for your S3 bucket.
+3. From the Mattermost distribution, upload the ``client`` directory to S3. Rename it to ``static``. You can use the AWS CLI command below from within the ``client`` directory. Note that the files are publicly readable, so be careful how you do it.
+
+    ``aws s3 cp --acl public-read --recursive . s3://static.spinmint.com/static/``
```

Comment:
Should there be a space before and after the "period" between "--recursive" and "s3"?

Reply:
Yup. `.` means current directory :P